src/mux-analytics.brs

```diff
@@ -60,6 +60,7 @@ function runBeaconLoop()
     m.top.video.ObserveField("state", m.messagePort)
     m.top.video.ObserveField("content", m.messagePort)
     m.top.video.ObserveField("control", m.messagePort)
+    m.top.video.ObserveField("streamingSegment", m.messagePort)
```

Comment:
This is backwards-compatible, right? As in if the firmware version doesn't support this, this essentially no-ops?

Reply:
This appears to be the case. observeField for non-existent fields seems to do nothing - setField on the same non-existent field does blow up.